packages/react-native/ReactAndroid/cmake-utils/default-app-setup/OnLoad.cpp

```diff
@@ -56,8 +61,10 @@ void registerComponents(
   REACT_NATIVE_APP_COMPONENT_REGISTRATION(registry);
 #endif
 
+#if AUTOLINKING_AVAILABLE
```

Comment:
I don't recall if we can enable New Arch at runtime on Android. I think on iOS, we can? Won't this break that?

Reply:
No, android can't disable the new arch at runtime. You can disable some pieces, but it is not recommended.